src/box/lua/load_cfg.lua

```diff
@@ -801,7 +801,7 @@ local function reconfig_modules(module_keys, oldcfg, newcfg, log_basecfg)
             oldvals = {}
             for key in pairs(keys) do
                 oldvals[key] = oldcfg[key]
-                rawset(oldcfg, key, newcfg[key])
+                oldcfg[key] = newcfg[key]
```

Comment:
Why all these three changes above are required? AFAIU, `oldcfg` parameter is `raw_cfg` as a result of your patch and this table has no metatable, hence the change has no effect. However, the code in `load_cfg.lua` is not quite clear, so I might be wrong.

Reply:
`rawset` is not required anymore because internally we start to use `raw_cfg`. So I would drop `rawset` too.

Reply:
Great, thanks!